templates/layout.html.twig

```diff
@@ -8,8 +8,8 @@
                 <div class="header-logo">
                     <a href="{{ path('app_homepage'|applyContext) }}">
                         <img class='logo' src="{{ preload(asset('build/images/logo.svg'), { as: 'image'}) }}"
-                             aria-label="Logo" height="45" width="45">
-                        <span>Koillection</span>
+                             aria-label="Logo" height="45" width="60">
+                        <span>Muzeum ETI</span>
```

Comment:
TODO: It remains the same in different languages.
Can we put `if` here in Twig or translate?

Reply:
It is possible to translate this using something like `<span>{{ 'global.eti-museum'|trans }}</span>` and adding the appropriate translation to the translation files.
Keep in mind that languages without that new translation will display just the untranslated translation key instead, so it would be best to add that translation to all languages currently supported by Koillection.

Reply:
Done.
Added appropriate translation to files for all languages.